### PR TITLE
PALS-40: AIW: Update "Group Unavailable" modal

### PR DESCRIPTION
**Pull request by crystalfinch**
Resolves PALS-40

## Description

Show "Please log in" heading and login button if user cannot access image group. User gets redirected back to the group they were looking at after logging in.

## Checks

**Have you written any necessary unit tests?**

- [ ] Yes
- [ ] Not yet
- [x] Not applicable

**Have you added or updated any necessary integration tests?**

- [ ] Yes
- [x] Not yet
- [ ] Not applicable

**Browsers tested on:**

- [x] Chrome
  - [ ] Mobile
- [ ] Safari
  - [ ] Mobile
- [ ] Firefox
  - [ ] Mobile
- [ ] Edge
- [ ] IE11
- [ ] Not applicable

### CODE REVIEW

File: src/app/modals/no-ig/no-ig.component.pug [hunk 1]
```diff
@@ -3,10 +3,12 @@
     // modal content: Is deleted, now what?
     .modal-content
       .modal-header
-        h4.modal-title([innerHtml]="'NO_IG.HEADING' | translate")
+        h4.modal-title(*ngIf="!noAccessIg", [innerHtml]="'NO_IG.HEADING_NO_EXISTS' | translate")
```
Comment (robert-hirsch-po) [inline src/app/modals/no-ig/no-ig.component.pug]: I would just make this a ternary in the innerHtml
`[innerHtml]="noAccessIg ? 'NO_IG.HEADING_NO_ACCESS' : 'NO_IG.HEADING_NO_EXISTS' | translate"`

File: src/app/modals/no-ig/no-ig.component.pug [hunk 2]
```diff
@@ -3,10 +3,12 @@
...
+        h4.modal-title(*ngIf="noAccessIg", [innerHtml]="'NO_IG.HEADING_NO_ACCESS' | translate")
       .modal-body
         p(*ngIf="!noAccessIg", [innerHtml]="'NO_IG.NO_EXISTS' | translate")
```
Comment (robert-hirsch-po) [inline src/app/modals/no-ig/no-ig.component.pug]: I would make this a ternary as well

File: src/app/modals/no-ig/no-ig.component.pug (hunk 1 above)
Comment (crystalfinch) [reply src/app/modals/no-ig/no-ig.component.pug]: Screen Shot 2020-03-12 at 7 22 57 PM
I can't seem to make this work. It just spits out the literal constant name. I tried wrapping the ternary in parentheses as well. :\

Comment (crystalfinch) [reply src/app/modals/no-ig/no-ig.component.pug]: image

Comment (crystalfinch) [reply src/app/modals/no-ig/no-ig.component.pug]: Ok, I was able to make it work but it's still ugly 😛

Comment (crystalfinch) [reply src/app/modals/no-ig/no-ig.component.pug]: Refactored in fa3dc38

File: src/app/modals/no-ig/no-ig.component.pug (hunk 2 above)
Comment (crystalfinch) [reply src/app/modals/no-ig/no-ig.component.pug]: Refactored in fa3dc38

**Pull request merged**